docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,356 @@
+# Installing With SR-IOV Compute Nodes
+
+## Table of Contents
+
+- [Pre-Install Requirements](#pre-install-requirements)
+- [Preparing your OpenShift Cluster to use SR-IOV Networks](#preparing-your-openshift-cluster-to-use-sr-iov-networks)
+- [Creating SR-IOV Networks for Compute Machines](#creating-sr-iov-networks-for-compute-machines)
+- [Creating SR-IOV Compute Nodes in IPI](#creating-sr-iov-compute-nodes-in-ipi)
+- [Creating SR-IOV Compute Nodes in UPI](#creating-sr-iov-compute-nodes-in-upi)
+
+## Pre-Install Requirements
```

Comment:
Pre-install requirements should probably just be a bulleted list. 
No need to sale the feature here. The user is already using it, thus the reason they are looking at the manual.

Reply:
This is just copy paste from downstream

Reply:
Where downstream?

Reply:
@iamemilio ^^^ Not sure how much of this is current. I couldn't find much of it in the latest downstream preview.